README.md

```diff
@@ -113,4 +114,4 @@ npm run build:zuora
 The minified Zuora stylesheet will then build to `/dist/honeycomb.css`.
 
 ## Contributing to the web toolkit
-To contribute to the web toolkit project, please create a Pull Request and let us know on the #honeycomb Slack channel. We'll then review the Pull Request and work with you to get it merged into master.
\ No newline at end of file
+To contribute to the web toolkit project, please create a Pull Request and let us know on the #honeycomb Slack channel. We'll then review the Pull Request and work with you to get it merged into master.
```

Comment:
not sure why this is in the diff because I didn't touch it, I even just used the editor within Github :o